From 4657320b2a3a817cc541feee4bfe0b68043e468c Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 7 Mar 2024 10:36:19 +0100
Subject: [PATCH 0003/1695] README.md: add security advisories.

Update README.md accordingly to the vulnerability management decided.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 2d2d6201a02ee0bc54bf02297ea4ee1cd9bf1a00 README.md: add security advisories.]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I96cdfed83a7f7d0380929b67f28d19d756df3436
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/365064
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
(cherry picked from commit 6a1c965c00d996946670419825b319f182187280)
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/370967
---
 README.md | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/README.md b/README.md
index 328078e1e..69ccfdcf3 100644
--- a/README.md
+++ b/README.md
@@ -4,4 +4,9 @@ project.
 
 All official OP-TEE documentation has moved to http://optee.readthedocs.io.
 
+Open-source OP-TEE has identified security advisories here
+https://github.com/OP-TEE/optee_os/security.
+Please consider the status of these advisories before using the ST OP-TEE
+project.
+
 // OP-TEE core maintainers
-- 
2.47.2

